### PR TITLE
updated registry to version 2.4.0 taged with 1.0.1

### PR DESCRIPTION
**Pull request by supermeng**
updated registry to version 2.4.0 taged with 1.0.1 and 
updated go with version 1.6

### CODE REVIEW

File: 1.0.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM sunyi00/centos-golang:3.0.0
```
Comment (sunyi00) [inline 1.0.1/Dockerfile]: 换用最新的 golang1.6.1 的那个 tag 吧？

Comment (sunyi00) [inline 1.0.1/Dockerfile]: 啊可能叫 go1.6.1

Comment (supermeng) [inline 1.0.1/Dockerfile]: soga,好滴

Comment (supermeng) [inline 1.0.1/Dockerfile]: 已经更新了

**Pull request merged**